### ISSUE TITLE
error 2345 0037

### ISSUE THREAD

**Issue by fallkord** [XorTroll/Goldleaf#596]
When i click to start goldleaf, the program crash. Fatal error 2345-0037

**Comment by EmreTech** [comment]
Same here. I'm not sure why.

**Comment by muddpunk** [comment]
I just started getting this today too (been fine for ages). The only change I have made is downloading MissionControl by ndeadly from the Homebrew App Store today so as I could use my spare Xbox controllers.

Goldleaf fatal report
-Goldleaf version: 0.9.0
-Current time 19:40:36
-Fatal result 0x4B59

When I try to run Goldleaf again I get:

Panic occurred while running Atmosphere.
Title ID 0100000000000004
Error: std::abort <0xFFE>

Press POWER to continue.

Current system version: 11.0.1|AMS M.18.1|S

**Comment by lookye** [comment]
MissionControl needs to be updated by its Dev. As long as this don´t happen, this error will happen to everyone using MissionControl and Goldleaf. 

@fallkord Do you have MissionControl on your sd?

**Comment by Borrajo** [comment]
I use both and don't have any problems.

Get Outlook para Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by lookye** [comment]
If youre using the **latest** AMS version, **latest** MissionControl + **latest** Goldleaf you will run into a crash. So dont tell me you dont have problems.

**Comment by ndeadly** [comment]
@lookye not true.

https://github.com/ndeadly/MissionControl/issues/220

**Comment by lookye** [comment]
I just tested it a minute ago. You cant start Goldleaf if MissionControl is active.

**Comment by ndeadly** [comment]
Read the thread. The presence of Mission Control will cause `sm` to abort, yes, but it's other outdated sysmodules that are to blame.

**Comment by lookye** [comment]
@ndeadly Youre right, I expanded my tests a little bit (took more time) and I need to correct myself.

1. Clean Ams + MissionControl + Goldleaf = no crash
2. Clean Ams + MissionControl + emuiibo + Goldleaf = no crash
3. Clean Ams + MissionControl + emuiibo + tesla + Goldleaf = no crash

So the problem lies somewhere else. If the people above would write down all the sysmodules they´re running it would help to track down the issue better.

**Comment by ndeadly** [comment]
From my experiences so far, it seems highly likely it's one of those I mentioned above. Of course there may still be others I don't know about. I believe the latest emuiibo release has been updated to work with atmosphere 0.18.0+, so that shouldn't cause problems as long as it's up to date. The latest ldn_mitm release however, while stated to work with the latest atmosphere, hasn't been updated since atmosphere 0.16.1. Atmosphere 0.18.0 made breaking changes for sysmodules that register services with `sm`, so I think it's just luck that it works at all. This would be my number one suspect.

If people want to post their `/atmosphere/contents` directories I'll happily take a look to see what's in there, or you can check it yourself against this list of sysmodule title IDs I compiled https://gist.github.com/ndeadly/a4b8c01bb453028cd0008f282098f696

**Comment by fallkord** [comment]
Sorry for no answer the questions. I suppose the problem was the sysmodule “sys-con”.
To resolve the problem i install AMS 18.1 clean + goldleaf lastest and no problems presents

**Comment by not-crazyramen** [comment]
> From my experiences so far, it seems highly likely it's one of those I mentioned above. Of course there may still be others I don't know about. I believe the latest emuiibo release has been updated to work with atmosphere 0.18.0+, so that shouldn't cause problems as long as it's up to date. The latest ldn_mitm release however, while stated to work with the latest atmosphere, hasn't been updated since atmosphere 0.16.1. Atmosphere 0.18.0 made breaking changes for sysmodules that register services with `sm`, so I think it's just luck that it works at all. This would be my number one suspect. Its TID is `4200000000000010`
> 
> If people want to post their `/atmosphere/contents` directories I'll happily take a look to see what's in there, or you can check it yourself against this list of sysmodule title IDs I compiled https://gist.github.com/ndeadly/a4b8c01bb453028cd0008f282098f696


Having the issue without sm in my `/atmosphere/contents` directory. Gonna try deleting them one by one to see which one does it.
